stomp_plugins/src/noise_generators/goal_guided_multivariate_gaussian.cpp

```diff
@@ -278,7 +288,7 @@ bool GoalGuidedMultivariateGaussian::generateNoise(const Eigen::MatrixXd& parame
   return true;
 }
 
-bool GoalGuidedMultivariateGaussian::generateRandomGoal(const Eigen::VectorXd& seed_joint_pose,Eigen::VectorXd& goal_joint_pose)
+bool GoalGuidedMultivariateGaussian::generateRandomGoal(const Eigen::VectorXd& reference_joint_pose,Eigen::VectorXd& goal_joint_pose)
```

Comment:
Hmm, I have no such thing as a "Constrained cartesian goal filter" in my configuration, should I?

Reply:
Hmm, so when I add `- class: stomp_moveit/ConstrainedCartesianGoal` as you have it in the example file it sometimes works (ca 60%) but there are still some failures whereas when effectively disabling the noise as in a few comments above it always works. (Except for cases where we compare to the "other" representation  of the euler angles). The right thing here would probably be to constrain the noise to be within the goal tolerances

Reply:
The **Constrained cartesian goal filter** enforces the goal tolerances.

Reply:
I also noticed that your `stomp_moveit/ToolGoalPose` cost function uses` 25.0` for the weights.  I've been using 0.5  for both
```
- class: stomp_moveit/ToolGoalPose
        position_cost_weight: 25.0
        orientation_cost_weight: 25.0
```
The plugin assumes that they add up to 1.   My fault for not documenting that

Reply:
Well I haven't checked that thoroughly this time, but last iteration the noise was too big so it would always be just outside the tolerances

maybe additionally you could add an error/warning if this does not add up to 1 (or normalize it). I will check on Monday if that changes anything